utils/docker/pull-or-rebuild-image.sh

```diff
@@ -120,11 +120,11 @@ for file in $files; do
 
 		# Check if the image has to be pushed to Docker Hub
 		# (i.e. the build is triggered by commits to the $GITHUB_REPO
-		# repository's master branch, and the Travis build is not
+		# repository's stable-1.6 branch, and the Travis build is not
 		# of the "pull_request" type). In that case, create the empty
 		# file.
 		if [[ "$TRAVIS_REPO_SLUG" == "$GITHUB_REPO" \
-			&& $TRAVIS_BRANCH == "master" \
+			&& $TRAVIS_BRANCH == "stable-1.6" \
```

Comment:
This part should more generic (branch like stable-* or branch = master), so we could stop updating this after each release.